certbot-apache/setup.py

```diff
@@ -11,7 +11,7 @@
     'acme=={0}'.format(version),
     'certbot=={0}'.format(version),
     'mock',
-    'python-augeas<=0.5.0',
+    'python-augeas',
```

Comment:
python-augeas 1.x is currently broken and that affects certbot tests: https://github.com/hercules-team/python-augeas/issues/26. Can certbot keep 0.5.0 for now?